content/blog/2023-10-24.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "2023-10-24"
+date = 2023-10-24
++++
+
+---
+
+Today is Tuesday, day 297 of 365 (81.36%). We're in week 43 of 52 (82.69%). 
+
+## Planning
+
+What I am going to do today: 
+
+- [x] [finish Flutter set up](https://docs.flutter.dev/get-started/editor?tab=vscode)
+
+## Activities executed
+
+Today I finished setting up Flutter and created a [git repository](https://github.com/OmnicodeSolutions/flutter_codelab) to work on the [flutter codelab](https://codelabs.developers.google.com/codelabs/flutter-codelab-first?hl=pt-br#0). I also updated [ansible-debian](https://github.com/OmnicodeSolutions/ansible-debian) to include the Flutter configuration.
+
+## Learnings
+
```

Comment:
You took more time trying to understand how to make flutter work, than to save it. You should register better this process. Why flutter wasn't working? What did you (or I) do to make it work? How did you install flutter?